Alert/alert.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert | UIElegance</title>
+    <link href="alert.css" rel="stylesheet">
+    <link href="../index.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <h1>UIElegance : Quick Useful UI Library!</h1>
+    </header>
+    <hr>
+    <div class="container ">
+        <aside>
+            <h2>Components</h2>
+            <ul>
+                <li><a href="/Avatar/avatar.html">Avatar</a></li>
+                <li><a class="current-page" href="/Alert/alert.html">Alert</a></li>
+                <li><a href="/Badge/badge.html">Badge</a></li>
+                <li><a href="/Button/button.html">Button</a></li>
+                <li><a href="/Card/card.html">Card</a></li>
+                <li><a href="/Image/image.html">Image</a></li>
+                <li><a href="/Input/input.html">Input</a></li>
+                <li><a href="/Lists/list.html">List</a></li>
+                <li><a href="/Navigation/navigation.html">Navigation</a></li>
+                <li><a href="/Modal/modal.html">Modal</a></li>
+                <li><a href="/Rating/rating.html">Rating</a></li>
+
+                <li><a href="/SimplifiedGrid/simplifiedGrid.html">Simplified Grid</a></li>
+                <li><a href="/TextUtilities/textutilities.html">Text Utilities</a></li>
+                <li><a href="/ColorPallet/colorpallet.html">Color Pallet</a></li>
+                <li><a href="/Toast/toast.html">Toast/Notify/Snackbar</a></li>
+                <li><a href="/index.html">Home</a></li>
+            </ul>
+        </aside>
+        <main>
+            <div class="component-main-display">
+                <h2>Alert</h2>
+                <p>Using the CSS class <Strong><code>.alert</code></Strong> you can highlight and bring important
+                    information or update to notice.</p>
+                <p>Copy below link into head tag of your HTML file:</p>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=night-owl&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=48px&ph=32px&ln=false&fl=1&fm=Hack&fs=13px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520href%253D%2522https%253A%252F%252Fgithub.com%252FRadhikaRJ%252FElegance%252Fblob%252Fmain%252FAlert%252Falert.css%2522%2520rel%253D%2522stylesheet%2522%253E"
+                    style="width: 838px; height: 187px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
border:0; transform: scale(1); overflow:hidden;

repetitive use of properties. These can be merged together inside one class name.